spec/features/courses/vacancies/edit_spec.rb

```diff
@@ -35,6 +36,12 @@
       course
     )
 
+    stub_request(
+      :post,
+      "http://localhost:3001/api/v2/providers/AO/courses/" \
+        "#{course_attributes[:course_code]}/sync_with_search_and_compare"
+    ).to_return(status: 201, body: "")
```

Comment:
Can we test that the controller action actually POSTs to this?

What about failure mode?

Reply:
Handle it in the card that covers validations.